### DISCUSSION TITLE
Any documentation on how to setup tls ?

### DISCUSSION THREAD

**Discussion by geminigeek** [cloudamqp/lavinmq#370] [Q&A] [answered]
hi,

the page with tls instruction is empty [https://lavinmq.com/documentation/amqps-tls](https://lavinmq.com/documentation/amqps-tls) , i guess you guys are in the process of writing docs, but is there any reference how to setup tls in lavinmq , is it similar to rabbitmq ?

**Comment by baelter** [comment]
Thanks for the reminder, will get to it asap!

**Comment by jage** [comment, marked as answer]
Docs added https://lavinmq.com/documentation/amqps-tls
